front/src/applications/stdcmV2/components/StdcmConfig.tsx

```diff
@@ -71,7 +69,11 @@ const StdcmConfig = ({
   const origin = useSelector(getOrigin);
   const destination = useSelector(getDestination);
 
-  const { pathfindingState } = usePathfinding(undefined, pathProperties);
+  const [pathfindingProperties, setPathfindingProperties] = useState<
+    ManageTrainSchedulePathProperties | undefined
```

Comment:
You can remove this undefined too
```suggestion
    ManageTrainSchedulePathProperties
```